### PR TITLE
Update info about renaming team site URL

### PR DESCRIPTION
**Pull request by rgsteele**
I wasn't able to verify whether the link to https://docs.microsoft.com/en-us/sharepoint/change-your-sharepoint-domain-name will work correctly, please check before merging

### CODE REVIEW

File: microsoft-365/admin/setup/domains-faq.yml [hunk 1]
```diff
@@ -179,7 +179,7 @@ sections:
             
           - **You can't rename the onmicrosoft domain after sign-up.** For example, if the initial domain you chose was fourthcoffee.onmicrosoft.com, you can't change it to be fabrikam.onmicrosoft.com. To use a different onmicrosoft.com domain, you'd have to start a new subscription with Microsoft 365. 
               
-          - **You can't rename your team site URL.** Your team site URL is based on your onmicrosoft.com domain name. Unfortunately, because of the way SharePoint Online architecture works, you can't rename the team site. 
+          - **You may not be able to change your SharePoint domain name.** Your SharePoint domain name is based on your onmicrosoft.com domain name. The ability to [change the SharePoint domain name](/sharepoint/change-your-sharepoint-domain-name.md) is available in public preview for organizations whose total SharePoint sites and OneDrive accounts number fewer than 1000.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/setup/domains-faq.yml]:
```suggestion
          - **You might not be able to change your SharePoint domain name.** Your SharePoint domain name is based on your onmicrosoft.com domain name. The ability to [change the SharePoint domain name](/sharepoint/change-your-sharepoint-domain-name) is available in public preview for organizations whose total SharePoint sites and OneDrive accounts number fewer than 1000.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @rgsteele, kindly commit the suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
@rgsteele Gentle reminder, please implement the suggested changes. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@pebaum Please merge this PR. Thanks

**Pull request merged**